### PR TITLE
Modify S6502: Improve description

### PR DESCRIPTION
**Pull request by loris-s-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [x] logical errors and incorrect information
- [x] information gaps and missing content
- [x] text style and tone
- [x] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S6502/docker/rule.adoc [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
+https://man7.org/linux/man-pages/man7/capabilities.7.html[capabilities] which
+is considered safe: this is called `sandbox` mode.
 
+If you disable the sandbox with the `--security=insecure` option, the command
+can use the full set of Linux capabilities. +
+This can lead to a container escape. For example, an attacker with the
+`SYS_ADMIN` capability is able to mount devices from the host system.
+
+This allows an attacker who controls the behavior of the ran command to access
+the host system, break out of the container and penetrate the infrastructure.
```
Comment (loris-s-sonarsource) [inline rules/S6502/docker/rule.adoc:15]:
```suggestion
the host system, break out of the container and penetrate the infrastructure.

After a successful intrusion, the underlying systems are exposed to:

* Theft of intellectual property and/or personal data
* Extortion
* Denial of services
```

File: rules/S6502/docker/rule.adoc [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
+`SYS_ADMIN` capability is able to mount devices from the host system.
 
+This allows an attacker who controls the behavior of the ran command to access
+the host system, break out of the container and penetrate the infrastructure.
+
+After a successful intrusion, the underlying systems are exposed to:
+
+* Theft of intellectual property and/or personal data
+* Extortion
+* Denial of services
```
Comment (loris-s-sonarsource) [inline rules/S6502/docker/rule.adoc:20]:
```suggestion
* Denial of services

```

File: rules/S6502/docker/rule.adoc [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
+`SYS_ADMIN` capability is able to mount devices from the host system.
+
+This allows an attacker who controls the behavior of the ran command to access
+the host system, break out of the container and penetrate the infrastructure.
+
+After a successful intrusion, the underlying systems are exposed to:
+
+* Theft of intellectual property and/or personal data
+* Extortion
+* Denial of services
```
Comment (loris-s-sonarsource) [inline rules/S6502/docker/rule.adoc:20]:
```suggestion
* Denial of service
```

**Review by hendrik-buchwald-sonarsource — APPROVED** [review]
LGTM in general, but I left some smaller comments.

File: rules/S6502/docker/rule.adoc [hunk 4]
```diff
@@ -1,11 +1,23 @@
...
+`SYS_ADMIN` capability is able to mount devices from the host system.
+
+This allows an attacker who controls the behavior of the ran command to access
+the host system, break out of the container and penetrate the infrastructure.
+
+After a successful intrusion, the underlying systems are exposed to:
+
+* Theft of intellectual property and/or personal data
+* Extortion
+* Denial of service
```
Comment (loris-s-sonarsource) [inline rules/S6502/docker/rule.adoc:20]:
```suggestion
* theft of intellectual property and/or personal data
* extortion
* denial of service
```

File: rules/S6502/docker/rule.adoc [hunk 5]
```diff
@@ -1,11 +1,23 @@
...
+By default, programs executed by a `RUN` statement use only a subset of
+https://man7.org/linux/man-pages/man7/capabilities.7.html[capabilities] which
+are considered safe: this is called `sandbox` mode.
 
+If you disable the sandbox with the `--security=insecure` option, the executed command
+can use the full set of Linux capabilities. +
+This can lead to a container escape. For example, an attacker with the
+`SYS_ADMIN` capability is able to mount devices from the host system.
+
+This allows an attacker who controls the behavior of the ran command to access
```
Comment (loris-s-sonarsource) [inline rules/S6502/docker/rule.adoc:13]:
```suggestion
This vulnerability allows an attacker who controls the behavior of the ran command to access
```

**Pull request merged**